service/core/action/post/list.go

```diff
@@ -69,7 +93,19 @@ func list(w http.ResponseWriter, r *http.Request) {
 
 	offset, limit := paginationx.Parse(r.URL.Query())
 
-	err = tx.Count(&result.Total).Order("id desc").Offset(offset).Limit(limit).Find(&posts).Error
+	if sort == "" {
+		sort = "asc"
+	}
```

Comment:
If sort is empty than it should be desc (latest ones)

Reply:
Okk